src/main/resources/application.properties

```
@@ -59,7 +57,8 @@ quarkus.http.auth.permission.authenticated.policy=authenticated
 %dev.quarkus.wiremock.devservices.reload=true
 %dev.quarkus.wiremock.devservices.files-mapping=src/test/resources/devservices/wiremock
 %dev.quarkus.rest-client.github.url=http://localhost:${quarkus.wiremock.devservices.port}/github
-%dev.quarkus.rest-client.morpheus.url=http://localhost:${quarkus.wiremock.devservices.port}/morpheus/scan
+#%dev.quarkus.rest-client.morpheus.url=http://localhost:${quarkus.wiremock.devservices.port}/morpheus/scan
+%dev.quarkus.rest-client.morpheus.url=https://agent-morpheus-rh-shared-morpheus.apps.ai-dev03.kni.syseng.devcluster.openshift.com/scan
```

@olavtar Please uncomment the wiremock route for responses stubs for dev profile, and comment the agent morpheus backend route ( this wiremock endpoint should be the default for dev profile, if needed to  actually test the changes in client in the backend, then it can be used and these two settings can be inverted).
```suggestion
%dev.quarkus.rest-client.morpheus.url=http://localhost:${quarkus.wiremock.devservices.port}/morpheus/scan
#%dev.quarkus.rest-client.morpheus.url=https://agent-morpheus-rh-shared-morpheus.apps.ai-dev03.kni.syseng.devcluster.openshift.com/scan
```